### PR TITLE
Docs: Add explanation for rotation layer

### PR DESCRIPTION
**Pull request by alfredo-d**
# What this PR does

Doc update to have explanation on rotation layer behavior for web schedule

## Which issue(s) this PR fixes
N/A

### CODE REVIEW

**Comment by Konstantinov-Innokentii** [comment]
@Matvey-Kuk could you please take a look?

File: docs/sources/calendar-schedules/web-schedule/create-schedule/index.md [hunk 1]
```diff
@@ -58,6 +58,10 @@ User groups that contain
 multiple users results in all users in the group being included in corresponding shifts.
 1. When you’re satisfied with the rotation preview, click **Create**.
 
+Oncall supports multiple **layer** for rotation which allows for overlapping schedule.
```
Comment (matiasb) [inline docs/sources/calendar-schedules/web-schedule/create-schedule/index.md:61]:
```suggestion
Oncall supports multiple **layers** for rotations which allow for overlapping shifts.
```

File: docs/sources/calendar-schedules/web-schedule/create-schedule/index.md [hunk 2]
```diff
@@ -58,6 +58,10 @@ User groups that contain
...
+Layer defines the schedule priority, for example *Layer 2* rotation override *Layer 1* rotation.
```
Comment (matiasb) [inline docs/sources/calendar-schedules/web-schedule/create-schedule/index.md:62]:
```suggestion
A layer defines the shift priority, for example a *Layer 2* shift overrides a *Layer 1* shift.
```

File: docs/sources/calendar-schedules/web-schedule/create-schedule/index.md [hunk 3]
```diff
@@ -58,6 +58,10 @@ User groups that contain
...
+In this case, users under *Layer 1* would not receive notification during the overlapping time with users under *Layer 2*.
```
Comment (matiasb) [inline docs/sources/calendar-schedules/web-schedule/create-schedule/index.md:63]:
```suggestion
In this case, users in a *Layer 1* shift will not be notified during the time overlapping with a shift in *Layer 2*.
```

**Review by matiasb — COMMENTED** [review]
Makes sense to add some documentation about this (see https://github.com/grafana/oncall/issues/1819). Suggested a few changes.

@Matvey-Kuk fyi, since you were working on some docs updates.

**Comment by alfredo-d** [comment]
Thanks @matiasb, I've committed your suggested changes.

**Review by matiasb — APPROVED** [review]

**Comment by Matvey-Kuk** [comment]
Need to be merged to dev, not main so closing.

**Pull request closed without merging**